src/test/java/K_pages/LoginPage.java

```diff
@@ -0,0 +1,43 @@
+package K_pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class LoginPage extends BasePage {
+    WebDriver driver;
+    WebDriverWait wait;
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )